portfolio-project-matching-app/tailwind.config.js

```diff
@@ -1,8 +1,27 @@
+const defaultTheme = require('tailwindcss/defaultTheme')
+
 module.exports = {
   purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      screens: {
+        'xs': '420px',
+        ...defaultTheme.screens,
+      },
+      colors: {
+        'custom-cool-extraDark': '#001219',
+        'custom-cool-dark': '#005f73',
+        'custom-cool-med': '#0a9396',
+        'custom-cool-light': '#94d2bd',
+        'custom-cool-extraLight': '#e9d8a6',
+        'custom-warm-extraLight': '#ee9b00',
+        'custom-warm-light': '#ca6702',
+        'custom-warm-med': '#bb3e03',
+        'custom-warm-dark': '#ae2012',
+        'custom-warm-darkExtra': '#9b2226'
```

Comment:
Here, we have imported the custom colors into Tailwind. As far as associating these colors with default styles e.g: `borderColorPrimary`, we have a few ways of going about things. We could establish these relationships in `styles/globals.css` or we could redundantly import the colors in `tailwind.config.js` and associate them with those descriptive names. I'm a bit more in favor of the former, but not sure which is the best approach.

Reply:
The main reason why I had hoped to keep the `tailwind.config.js` file similar to how it is setup currently is the following:
Suppose we decide that we want to change our color palette up a bit. In that case, we have about 10 hard-coded strings to manually update (assuming we don't add more colors to the range). Then, in the `.css` file all of our `borderColorPrimary: 'custom-cool-med'` would update automatically. In the other case, we would have to manually update all of those details.

All that said, I could be missing something big. Realistically, our project probably won't switch up palettes, so I think all this more with a larger app in mind. Honestly, I could go for any approach!

Reply:
With the size of our color palette, this seems like something we can leave as is for now and revisit at a future point. I can create an issue on the Project Board so we don't forget about it.